From 347f1493320e1bc2194c70d4d66bfe2b5883bf1e Mon Sep 17 00:00:00 2001
From: Kenneth Graunke <kenneth@whitecape.org>
Date: Mon, 30 Dec 2013 07:19:36 +0000
Subject: Revert "mesa: Remove GLXContextID typedef from glx.h."

This reverts commit 136a12ac98868d82c2ae9fcc80d11044a7ec56d1.

According to belak51 on IRC, this commit broke Allegro, which would no
longer compile.  Applications apparently expect the GLXContextID typedef
to exist in glx.h; removing it breaks them.  A bit of searching around
the internet revealed other complaints since upgrading to Mesa 10.

Cc: "10.0" <mesa-stable@lists.freedesktop.org>
(cherry picked from commit f425d56ba41382be04366d011536ee78a03a2f33)
---
diff --git a/include/GL/glx.h b/include/GL/glx.h
index 62d0ede..234abc0 100644
--- a/include/GL/glx.h
+++ b/include/GL/glx.h
@@ -168,6 +168,7 @@ typedef XID GLXDrawable;
 /* GLX 1.3 and later */
 typedef struct __GLXFBConfigRec *GLXFBConfig;
 typedef XID GLXFBConfigID;
+typedef XID GLXContextID;
 typedef XID GLXWindow;
 typedef XID GLXPbuffer;
 
--
cgit v0.9.0.2-2-gbebe

